src/db.js

```diff
@@ -26,9 +26,22 @@ const sequelize = new Sequelize({
   host: POSTGRES_HOST || 'localhost',
   dialect: 'postgres',
   port: POSTGRES_PORT || 5432,
-  password: POSTGRES_PASSWORD || '123',
+  password: POSTGRES_PASSWORD || '314159',
```

Comment:
Consider removing the default password or ensuring it is securely managed through environment variables to avoid security risks.